features/api_configure_retry_service.feature

```diff
@@ -0,0 +1,59 @@
+Feature: api.u.pro.attach.auto.configure_retry_service
+
+    @series.lts
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: v1 successfully triggers retry service when run during startup
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I change contract to staging with sudo
+        When I create the file `/lib/systemd/system/apitest.service` with the following
```

Comment:
No lock conflict should be happening.

This test is for running the `configure_retry_service`, which deserves a better explanation and documentation. I'll include that as soon as the api docs PR is in `main`.

Right now, the `configure_retry_service` function is written such that if it is called before `ubuntu-advantage.service` runs, then `ubuntu-advantage.service` will go into "retry" mode. `configure_retry_service` does not do anything that actually causes `ubuntu-advantage.service` to start (no `systemctl start ubuntu-advantage.service` and no path file or anything like that). 

That is because the intended use case is for cloud-init to call during boot before the `ubuntu-advantage.service` ConditionPathExists are checked. That is also why I named it `configure_retry_service` instead of `start_retry_service`.

Reply:
Got it and totally agree that we can a doc explanation for it

Reply:
Added!